DXRModules/DeusEx/Classes/DXREvents.uc

```diff
@@ -318,12 +346,21 @@ function SetWatchFlags() {
         break;
     case "15_AREA51_BUNKER":
         WatchFlag('JockBlewUp');
+        WatchFlag('blast_door_open');
         Tag = 'Area51FanShaft';
         foreach AllActors(class'ZoneInfo', zone) {
             if (zone.Tag=='fan'){
                 zone.ZonePlayerEvent = 'Area51FanShaft';
             }
         }
+
+        //This flag trigger actually doesn't trigger because a security computer can only trigger DeusExMovers
+        foreach AllActors(class'FlagTrigger',fTrigger,'blast_door'){
+            fTrigger.Tag = 'blast_door_flag';
+        }
+        foreach AllActors(class'DeusExMover',dxm,'blast_door'){
```

Comment:
Should probably make this work with HXMover, there's a preprocessor definition for it